index.js

```diff
@@ -1,24 +1,33 @@
 const mongoose = require('mongoose');
 
+
 // Import of the model Recipe from './models/Recipe.model.js'
 const Recipe = require('./models/Recipe.model');
 // Import of the data from './data.json'
 const data = require('./data');
 
-const MONGODB_URI = 'mongodb://localhost:27017/recipe-app';
+const MONGODB_URI = 'mongodb://127.0.0.1:27017/recipe-app';
 
 //Method 1 : Using Async Await
 
 const manageRecipes = async () => {
   try {
-    // Connection to the database "recipe-app"
     const dbConnection = await mongoose.connect(MONGODB_URI);
     console.log(`Connected to the database: "${dbConnection.connection.name}"`);
-
+    
     // Before adding any recipes to the database, let's remove all existing ones
     await Recipe.deleteMany();
-
-    // Run your code here, after you have insured that the connection was made
+    
+    await Recipe.create({title:"Bacalhau Natas"})
+    await Recipe.insertMany(data)
+    data.forEach(recipes=>console.log(recipes.title))
+      
```

Comment:
Here you are console logging all the recipe's title from the `data.json` file. But the purpose of this step was to console.log it through the `database`:

```
    const manyRecipes = await Recipe.insertMany(data);
    manyRecipes.forEach((recipe) =>
      console.log(`New recipe added: ${recipe.title}`)
    );
```